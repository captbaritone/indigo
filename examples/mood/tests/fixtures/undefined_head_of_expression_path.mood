pub fn test(): f64 {
  Oops::Bar
}
