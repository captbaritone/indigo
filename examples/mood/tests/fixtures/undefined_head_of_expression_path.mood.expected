Error: Undefined enum "Oops":
 --> undefined_head_of_expression_path.mood:2:3

  | pub fn test(): f64 {
2 |   Oops::Bar
  |   ^^^^ "Oops" is not defined.
  | }
